_config.e2e_po_search_enabled.yml

```diff
@@ -26,7 +26,4 @@ defaults:
       lang: fr
   - scope:
```

Comment:
I think we can remove this whole block `scope:` if we're no longer overriding any values.

Reply:
resolved with commit eeee6e5a1efc4a9913f17280c8bcde4672681974